references/classification/transforms.py

```diff
@@ -13,13 +13,13 @@ def get_mixup_cutmix(*, mixup_alpha, cutmix_alpha, num_categories, use_v2):
     mixup_cutmix = []
     if mixup_alpha > 0:
         mixup_cutmix.append(
-            transforms_module.MixUp(alpha=mixup_alpha, num_categories=num_categories)
+            transforms_module.MixUp(alpha=mixup_alpha, num_classes=num_categories)
```

Comment:
Thanks a lot for the report and for the fix @zhulinchng . Can you also change the signature of `get_mixup_cutmix` to accept a `num_classes` parameter instead of `num_categories`?